src/copmressionFile.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
```

Comment:
The file name 'copmressionFile.js' contains a typo. Consider renaming it to 'compressionFile.js' to match the exported function.